Resources/T-Tests/qml/TTestBayesianIndependentSamples.qml

```diff
@@ -82,11 +82,11 @@ Form {
 		title: qsTr("Tests")
 		RadioButton
 		{
-            id: student
+			id: student
 			value: "Student";	label: qsTr("Student"); checked: true }
 		RadioButton
 		{
-			value: "Wilcoxon";	label: qsTr("Mann-Whitney");
+			value: "Wilcoxon";	label: qsTr("Mann-Whitney"); id: testWilcoxon
```

Comment:
why the new id?

Reply:
I wanted to disable the SetSeed component if Wilcoxon wasn't selected (the Student t-test doesn't require seed), forget to add it to the new component in the rewrite. The question is, do we wanna have the option to set seed disabled if it doesn't impact the analysis?

Reply:
ok, I found out that I missed sampling for the prior/posterior plot, so, the seed is needed for all the t-tests anyway